src/lib389/lib389/replica.py

```diff
@@ -2362,13 +2365,16 @@ def ensure_agreement(self, from_instance, to_instance, init=False):
         assert dn is not None
         assert creds is not None
 
+        timeout = 5
+        timeout = int(timeout * get_timeout_scale())
```

Comment:
I think `round()` will give more correct results.

Like, if DS_TIMEOUT_SCALE=1.6 and `timeout` is `6`, then:
- float is 9.6
- `int(timeout * get_timeout_scale()) == 9`
- `round(timeout * get_timeout_scale()) == 10`

Reply:
Sure, I'm okay with both options. I just thought that the intention was to round the number. If we are okay with just discarding any remainder, I see no issues with using `int()`.

Reply:
Ok, I replaced it with `round()` everywhere.